# Copyright (C) 1999-2001 Free Software Foundation, Inc.
# Valek Filippov <frob@df.ru>, 1999-2001.
#
msgid ""
msgstr ""
"Project-Id-Version: libgda\n"
"POT-Creation-Date: 2001-12-29 00:08+0100\n"
"PO-Revision-Date: 2001-10-13 21:14+0400\n"
"Last-Translator: Valek Filippov <frob@df.ru>\n"
"Language-Team: Russian <ru@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=koi8-r\n"
"Content-Transfer-Encoding: 8-bit\n"

#: libgda/gda-client.c:200 libgda/gda-client.c:294
#, fuzzy, c-format
msgid "Data source %s not found in configuration"
msgstr "База данных '%s' не найдена в конфигурации системы"

# fe/fe_log.c:92
#: libgda/gda-client.c:221
#, fuzzy, c-format
msgid "Could not activate object %s"
msgstr "Не удалось создать файл %s"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: libgda/gda-client.c:239
#, fuzzy
msgid "Could not create connection component"
msgstr "не удалось открыть соединение с %s"

#: libgda/gda-config.c:515
#, fuzzy
msgid "Could not query CORBA components"
msgstr "Не удалось открыть фабрику COBRA"

#: libgda/gda-config.c:622
#, fuzzy
msgid "Could not query CORBA providers"
msgstr "Не удалось открыть фабрику COBRA"

#: libgda/gda-config.c:835
msgid "Name"
msgstr ""

#: libgda/gda-config.c:836
msgid "Provider"
msgstr ""

#: libgda/gda-config.c:837
#, fuzzy
msgid "Connection string"
msgstr "Соединение было закрыто"

#: libgda/gda-config.c:838
msgid "Description"
msgstr ""

#: libgda/gda-config.c:839
msgid "Username"
msgstr ""

#: libgda/gda-error.c:127
#, c-format
msgid "%s: An Error occured in the CORBA system."
msgstr "%s: Возникла ошибка в системе CORBA."

#: libgda/gda-error.c:156
msgid "Unknown CORBA exception for connection"
msgstr "Неизвестное исключение для соединения CORBA"

#: libgda/gda-init.c:44
msgid "Attempt to initialize an already initialized client"
msgstr "Попытка инициализировать уже инициализированного клиента"

#: libgda/gda-init.c:56
msgid "Could not initialize Bonobo"
msgstr "Не удалось инициализировать Bonobo"

#: libgda/gda-server.c:95
#, c-format
msgid "Don't know how to create components with IID %s"
msgstr ""

# fe/fe_log.c:92
#: libgda/gda-server.c:102
#, fuzzy, c-format
msgid "Could not create component of type %s"
msgstr "Не удалось создать файл %s"

#: libgda/gda-server-connection.c:90
msgid "Could not duplicate client object. Client won't get notifications"
msgstr ""

#: libgda/gda-server-connection.c:440
#, fuzzy, c-format
msgid "Could not notify client about action %d"
msgstr "Не удалось получить клиентское сообщение # %d"

#: libgda/gda-server-recordset.c:204
msgid "No implementation for the fetch() method found"
msgstr ""

# gda-clnt/gda-field.c:117
#: libgda/gda-value.c:573
msgid "TRUE"
msgstr "ИСТИНА"

# gda-clnt/gda-field.c:119
#: libgda/gda-value.c:575
msgid "FALSE"
msgstr "ЛОЖЬ"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: libgda/gda-xml-database.c:162
#, fuzzy, c-format
msgid "Could not load file at %s"
msgstr "Не удалось открыть файл %s"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: libgda/gda-xml-database.c:173
#, fuzzy, c-format
msgid "Could not parse file at %s"
msgstr "Не удалось открыть файл %s"

#: libgda/gda-xml-database.c:183
#, fuzzy, c-format
msgid "Bad formed document at %s"
msgstr "плохо сформированный документ"

#: libgda/gda-xml-database.c:258
#, c-format
msgid "table %s already exists"
msgstr "таблица %s уже существует"

#: libgda/gda-xml-database.c:440
#, c-format
msgid "%p is not a valid table"
msgstr "%p не является допустимой таблицей"

#: providers/default/gda-default-provider.c:141
msgid ""
"A full path must be specified on the connection string to open a database."
msgstr ""

#: providers/default/gda-default-provider.c:199
msgid "Invalid DEFAULT handle"
msgstr ""

#: providers/default/GNOME_Database_Provider_Default.server.in.h:1
#, fuzzy
msgid "Default component factory"
msgstr "Не удалось открыть фабрику COBRA"

#: providers/default/GNOME_Database_Provider_Default.server.in.h:2
msgid "GDA Default Datasource Access"
msgstr "GDA доступ к источнику данных по умолчанию"

#: providers/default/main.c:47
#, fuzzy
msgid "Could not initiate Default component factory"
msgstr "Не удалось зарегистрировать реализацию провайдера по умолчанию"

#: providers/mysql/gda-mysql-provider.c:171
#, fuzzy
msgid ""
"You cannot provide a UNIX_SOCKET if you also provide either a HOST or a PORT."
msgstr ""
"Вы не можете указывать UNIX_SOCKET, если вы также указываете Хост или Порт. "
"Пожалуйста удалите параметры Хост и Порт или UNIX_SOCKET"

#: providers/mysql/gda-mysql-provider.c:242
#: providers/mysql/gda-mysql-provider.c:321
#: providers/mysql/gda-mysql-provider.c:350
#: providers/mysql/gda-mysql-provider.c:379
msgid "Invalid MYSQL handle"
msgstr ""

#: providers/mysql/GNOME_Database_Provider_MySQL.server.in.h:1
msgid "GDA Datasource Access for MySQL"
msgstr "GDA доступ к источнику данных для MySQL"

#: providers/mysql/GNOME_Database_Provider_MySQL.server.in.h:2
msgid "MySQL component factory"
msgstr ""

#: providers/mysql/main.c:48
#, fuzzy
msgid "Could not initiate MySQL component factory"
msgstr "Не удалось зарегистрировать реализацию провайдера MySQL"

#: providers/sybase/GNOME_Database_Provider_Sybase.server.in.h:1
msgid "GDA Datasource Access for Sybase"
msgstr "GDA доступ к источнику данных для Sybase"

#: providers/sybase/GNOME_Database_Provider_Sybase.server.in.h:2
msgid "Sybase component factory"
msgstr ""

#: providers/sybase/gda-sybase-error.c:137
msgid "ct_diag 1 failed in sybase_check_clientmsg"
msgstr ""

#: providers/sybase/gda-sybase-error.c:159
#: providers/sybase/gda-sybase-error.c:197
msgid "ct_diag 2 failed in sybase_check_clientmsg"
msgstr ""

#: providers/sybase/gda-sybase-error.c:167
#: providers/sybase/gda-sybase-error.c:376
#, fuzzy
msgid "Client Message: "
msgstr "Сообщение клиента:"

#: providers/sybase/gda-sybase-error.c:178
#: providers/sybase/gda-sybase-error.c:387
#, fuzzy
msgid "Informational Client  Message: "
msgstr "Сообщение клиента:"

#: providers/sybase/gda-sybase-error.c:239
msgid "ct_diag 1 failed in sybase_check_servermsg"
msgstr ""

#: providers/sybase/gda-sybase-error.c:261
#: providers/sybase/gda-sybase-error.c:304
msgid "ct_diag 2 failed in sybase_check_servermsg"
msgstr ""

#: providers/sybase/gda-sybase-error.c:274
#, fuzzy
msgid "Server Message: "
msgstr "Сообщение сервера:"

#: providers/sybase/gda-sybase-error.c:285
#, fuzzy
msgid "Informational Server  Message: "
msgstr "Сообщение сервера:"

#: providers/sybase/gda-sybase-error.c:346
msgid "cs_diag 1 failed in sybase_check_clientmsg"
msgstr ""

#: providers/sybase/gda-sybase-error.c:368
#: providers/sybase/gda-sybase-error.c:406
msgid "cs_diag 2 failed in sybase_check_clientmsg"
msgstr ""

#: providers/sybase/gda-sybase-error.c:429
msgid "SYB"
msgstr ""

#: providers/sybase/gda-sybase-error.c:431
msgid "severity"
msgstr ""

#: providers/sybase/gda-sybase-error.c:433
msgid "number"
msgstr ""

#: providers/sybase/gda-sybase-error.c:435
msgid "origin"
msgstr ""

#: providers/sybase/gda-sybase-error.c:437
msgid "layer"
msgstr ""

#: providers/sybase/gda-sybase-error.c:444
#, c-format
msgid ""
"\n"
"\tOperating system error number(%ld):\n"
"\t%s"
msgstr ""

#: providers/sybase/main.c:49
#, fuzzy
msgid "Could not initiate Sybase component factory"
msgstr "Не удалось открыть фабрику COBRA"

# fe/fe_main.c:205
#~ msgid "Data source %s not found"
#~ msgstr "Не найден источник данных %s"

#~ msgid "[GDA Client Library]"
#~ msgstr "[GDA клиентская библиотека]"

#~ msgid "CORBA System exception: %s"
#~ msgstr "Исключение системы CORBA: %s"

#~ msgid "True"
#~ msgstr "Истина"

# fe/fe_main.c:249
#~ msgid "False"
#~ msgstr "Ложь"

#~ msgid "[CORBA User Exception]"
#~ msgstr "[Исключение пользователя CORBA]"

# gda-clnt/gda-field.c:95
#~ msgid "<NULL>"
#~ msgstr "<NULL>"

# gda-clnt/gda-field.c:101 gda-clnt/gda-field.c:103
#~ msgid "<Unknown GDA Type(NULL)>"
#~ msgstr "<Неизвестный GDA тип(NULL)>"

#~ msgid "Saving log...x"
#~ msgstr "Сохранение журнала...x"

#~ msgid "ERROR: "
#~ msgstr "ОШИБКА:"

#~ msgid "MESSAGE: "
#~ msgstr "СООБЩЕНИЕ:"

# fe/fe_main.c:205
#~ msgid "Item with id %s not found"
#~ msgstr "Не найден элемент с идентификатором %s"

#~ msgid "Could not get pointer to server implementation"
#~ msgstr "Невозможно получить указатель на реализацию сервера"

#~ msgid "Received signal %d, shutting down"
#~ msgstr "Получен сигнал %d, выключение"

#~ msgid "Received signal %d. Ignoring"
#~ msgstr "Получен сигнал %d. Игнорируется"

#~ msgid "Attempt to initialize an already initialized provider"
#~ msgstr "Попытка инициализации уже инициализированного провайдера"

#~ msgid "%s not implemented"
#~ msgstr "%s не реализовано"

#~ msgid "Starting provider %s with no implementation functions"
#~ msgstr "Запуск провайдера %s с не реализованными функциями"

#~ msgid "CORBA system exception %s"
#~ msgstr "Исключение системы CORBA %s"

#~ msgid "CORBA user exception: %s"
#~ msgstr "Исключение пользователя CORBA %s"

#~ msgid "You must specify a directory for the database to be stored"
#~ msgstr "Вы должны указать каталог, в котором будет сохранена база данных"

#~ msgid "Unknown schema type"
#~ msgstr "Неизвестный тип схемы"

#~ msgid "GDA Datasource Access for LDAP"
#~ msgstr "GDA доступ к источнику данных LDAP"

#~ msgid "error '%s' at %s"
#~ msgstr "ошибка '%s' в %s"

#~ msgid "Not available"
#~ msgstr "Не доступно"

#~ msgid "error"
#~ msgstr "ошибка"

#~ msgid "Could not register LDAP provider implementation"
#~ msgstr "Не удалось зарегистрировать реализацию провайдера LDAP"

#~ msgid "GDA Datasource Access for Interbase"
#~ msgstr "GDA доступ к источнику данных для Interbase"

#~ msgid "Error code %ld"
#~ msgstr "Код ошибки %ld"

#~ msgid "Could not register INTERBASE provider implementation"
#~ msgstr "Не удалось зарегистрировать реализацию провайдера INTERBASE"

#~ msgid "GDA Datasource Access for Mail"
#~ msgstr "GDA доступ к источнику данных для Mail"

#~ msgid "Unhandled SCHEMA_QTYPE %d"
#~ msgstr "Необработанный SCHEMA_QTYPE %d"

#~ msgid "Could not register Mail provider implementation"
#~ msgstr "Не удалось зарегистрировать реализацию провайдера Mail"

#~ msgid "GDA Datasource Access for MDB (MS Access) files"
#~ msgstr "GDA доступ к источнику данных для файлов MDB (MS Access)"

# fe/fe_editor.c:185
#~ msgid "executing command '%s'"
#~ msgstr "выполнение команды \"%s\""

# fe/fe_editor.c:185
#~ msgid "info '%s'"
#~ msgstr "информация \"%s\""

#~ msgid ""
#~ "%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST "
#~ "or a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and "
#~ "PORT options"
#~ msgstr ""
#~ "%s:%d: Вы не можете указывать UNIX_SOCKET, если вы также указываете Хост "
#~ "или Порт. Пожалуйста удалить параметры Хост и Порт или UNIX_SOCKET"

#~ msgid "%s:%d: You must  provide the filename with the MS Access database"
#~ msgstr "%s:%d: Вы должны ввести имя файла базы данных MS Access"

#~ msgid "Opening connection with filename=%s, user=%s, password=%s"
#~ msgstr ""
#~ "Открытие соединения с параметрами имя_файла=%s, пользователь=%s, пароль=%s"

#~ msgid "mysql_recset is NULL"
#~ msgstr "mysql_recset имеет значение NULL"

#~ msgid "No table name given for SCHEMA_COLS"
#~ msgstr "Не задано название таблицы для SCHEMA_COLS"

#~ msgid ""
#~ "Opening connection with user=%s, password=%s, host=%s, port=%s, "
#~ "unix_socket=%s"
#~ msgstr ""
#~ "Открытие соединения с параметрами пользователь=%s, пароль=%s, хост=%s, "
#~ "порт=%s, unix_socket=%s"

#~ msgid "Unknown"
#~ msgstr "Неизвестно"

#~ msgid "GDA Datasource Access for ODBC"
#~ msgstr "GDA доступ к источнику данных для ODBC"

#~ msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
#~ msgstr "gda_odbc_connection_open_schema: необработанный SCHEMA_QTYPE %d\n"

#~ msgid "no text"
#~ msgstr "нет текста"

#~ msgid "Could not register ODBC provider implementation"
#~ msgstr "Не удалось зарегистрировать реализацию провайдера ODBC"

#~ msgid "GDA Datasource Access for Oracle"
#~ msgstr "GDA доступ к источнику данных для Oracle"

# fe/fe_log.c:92
#~ msgid "Could not allocate statement handle"
#~ msgstr "Не удалось получить обработчик утверждения"

#~ msgid "command object being created without associated connection"
#~ msgstr "командный объект был создан без ассоциированного соединения"

# fe/fe_editor.c:185
#~ msgid "Executing command '%s'"
#~ msgstr "Выполнение команды \"%s\""

#~ msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
#~ msgstr "ошибка в OCIHandleAlloc() HTYPE_SVCCTX"

#~ msgid "error in OCIHandleAlloc() HTYPE_SERVER"
#~ msgstr "ошибка в OCIHandleAlloc() HTYPE_SERVER"

#~ msgid "error in OCIHandleAlloc() HTYPE_ERROR"
#~ msgstr "ошибка в OCIHandleAlloc() HTYPE_ERROR"

#~ msgid "Unknown ORACLE type %ld"
#~ msgstr "Неизвестный тип ORACLE %ld"

#~ msgid "Unknown ORACLE data type in fill_field_values"
#~ msgstr "Неизвестный ORACLE тип данных в fill_field_values"

#~ msgid "recordset object being created without associated connection"
#~ msgstr "объект recordset был создан без ассоциированного соединения"

#~ msgid "Could not register ORACLE provider implementation"
#~ msgstr "Не удалось зарегистрировать реализацию провайдера ORACLE"

#~ msgid "GDA Datasource Access for Postgres"
#~ msgstr "GDA доступ к источнику данных для Postgres"

#~ msgid "postgres_recset is NULL"
#~ msgstr "postgres_recset имеет значение NULL"

#~ msgid "Could not register PostgreSQL provider implementation"
#~ msgstr "Не удалось зарегистрировать реализацию провайдера PostgreSQL"

#~ msgid "Connection seems to be dead"
#~ msgstr "Видимо соединение умерло"

#~ msgid "Filling command structure failed\n"
#~ msgstr "Сбой заполнения управляющей структуры\n"

#~ msgid "sending cmd failed"
#~ msgstr "сбой отправки кмд"

#~ msgid "Command execution finished, all results processed"
#~ msgstr "Выполнение команды завершено, все результаты обрабатываются"

# fe/fe_editor.c:138
#~ msgid "Command failed"
#~ msgstr "Выполнение команды не удалось"

#~ msgid "Execution of resultless command succeeded"
#~ msgstr "Выполнение \"безрезультатной\" команды прошло успешно"

#~ msgid "ct_results() returned %x"
#~ msgstr "ct_results() вернуло %x"

#~ msgid "No further results, command completly succeeded."
#~ msgstr "Нет последующих результатов, команда выполнена успешно."

#~ msgid "An error occured during command execution."
#~ msgstr "При выполнении команды возникла ошибка."

#~ msgid "Cmd execution returned unknown result, may be an error"
#~ msgstr ""
#~ "Выполнение команды привело к неизвестному результату, возможно к ошибке"

#~ msgid "No data returned, freeing recordset structure"
#~ msgstr "Данные не были возвращены, освобаждается структура записи"

#~ msgid "Freeing command structure failed"
#~ msgstr "Сбой освобождения структуры команды"

#~ msgid "Not enough memory for sybase_Connection"
#~ msgstr "Недостаточно памяти для выполнения sybase_Соединения"

#~ msgid "schema_cols: table_name '%s'"
#~ msgstr "schema_cols: название таблицы \"%s\""

# fe/fe_log.c:92
#~ msgid "Could not allocate command structure"
#~ msgstr "Не удалось получить структуру команды"

# fe/fe_log.c:92
#~ msgid "Could not allocate enough memory for query"
#~ msgstr "Не удалось получить достаточно памяти для запроса"

#~ msgid "schema_procs: invalid constraint type %d"
#~ msgstr "schema_procs: недопустимый тип ограничения %d"

# fe/fe_log.c:92
#~ msgid "Could not allocate cmd structure"
#~ msgstr "Не удалось получить структуру кмд"

#~ msgid "schema_tables: invalid constraint type %d"
#~ msgstr "schema_tables: недопустимый тип ограничения %d"

#~ msgid "Invalid or unsupported constraint type: %d"
#~ msgstr "Неправильный или неподдерживаемый тип ограничения: %d"

# fe/fe_log.c:92
#~ msgid "Could not allocate GdaServerCommand"
#~ msgstr "Не удалось получить GdaServerCommand"

#~ msgid "connection status: %x"
#~ msgstr "состояние соединения: %x"

#~ msgid "return code: %x"
#~ msgstr "код возврата: %x"

# fe/fe_connection.c:88
#~ msgid "attempting reconnection"
#~ msgstr "попытка пересоединения"

# fe/fe_connection.c:46
#~ msgid "Reconnected to '%s'"
#~ msgstr "Пересоединение с \"%s\""

# fe/fe_connection.c:46
#~ msgid "Connected to %s"
#~ msgstr "Соединен с %s"

# fe/fe_designer.c:40
#~ msgid "Selected database '%s'"
#~ msgstr "Выбранная база данных \"%s\""

# fe/fe_log.c:92
#~ msgid "%s: Could not allocate cmd structure"
#~ msgstr "%s: Не удалось получить структуру кмд"

#~ msgid "%s: Could not allocate memory for cmd string"
#~ msgstr "%s: Не удалось получить память для строки кмд"

# fe/fe_editor.c:138
#~ msgid "%s: ct_command failed"
#~ msgstr "%s: сбой ct_command"

# fe/fe_editor.c:138
#~ msgid "%s: sending command failed"
#~ msgstr "%s: отправка команды не удалась"

#~ msgid "%s: selecting database failed"
#~ msgstr "%s: сбой выбора базы данных"

# fe/fe_editor.c:138
#~ msgid "%s: use command failed"
#~ msgstr "%s: сбой использования команды"

# fe/fe_log.c:92
#~ msgid "Could not allocate locale structure"
#~ msgstr "Не удалось получить структуру locale"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#~ msgid "Could not set locale to '%s'"
#~ msgstr "Не удалось установить локализацию для '%s'"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#~ msgid "Could not configure locale to be '%s'"
#~ msgstr "Не удалось настроить локализацию на '%s'"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#~ msgid "Could not drop locale structure"
#~ msgstr "Не удалось сбросить структуру locale"

#~ msgid "Fatal error: %s\n"
#~ msgstr "Фатальная ошибка: %s\n"

#~ msgid "gda_sybase_error_make(%p,%p,%p,%p)"
#~ msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#~ msgid "cs_diag() initialized"
#~ msgstr "cs_diag() инициализовано"

#~ msgid "Could not initialize cs_diag()"
#~ msgstr "Не удалось инициализировать cs_diag()"

#~ msgid "ct_diag() initialized"
#~ msgstr "ct_diag() инициализовано"

#~ msgid "Could not initialize ct_diag()"
#~ msgstr "Не удалось инициализировать ct_diag()"

#~ msgid "%s could not find server implementation"
#~ msgstr "%s невозможно найти реализацию сервера"

#~ msgid "Could not get # of messages"
#~ msgstr "Не удалось получить число сообщений"

#~ msgid "Could not get # of server messages"
#~ msgstr "Невозможно получить номер сообщения сервера"

#~ msgid "Fetching %d client messages"
#~ msgstr "Получение %d клиентских сообщений"

#~ msgid "Fetching %d client message"
#~ msgstr "Получение %d клиентского сообщения"

#~ msgid "Could not get # of client messages"
#~ msgstr "Не удалось получить число клиентских сообщений"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#~ msgid "Could not clear client messages"
#~ msgstr "Не удалось очистить клиентские сообщения"

#~ msgid "Fetching %d server messages"
#~ msgstr "Получение %d серверных сообщений"

#~ msgid "Fetching %d server message"
#~ msgstr "Получение %d серверного сообщения"

# fe/fe_log.c:92
#~ msgid "Could not get server msg # %d"
#~ msgstr "Не удалось получить серверное сообщение # %d"

# fe/fe_log.c:92
#~ msgid "Could not clear server messages"
#~ msgstr "Не удалось очистить серверные сообщения"

#~ msgid "Could not get # of cslib messages"
#~ msgstr "Не удалось получить число сообщений cslib"

#~ msgid "Fetching %d cslib messages"
#~ msgstr "Получение %d сообщений cslib"

#~ msgid "Fetching %d cslib message"
#~ msgstr "Получение %d сообщения cslib"

#~ msgid "Could not get cslib msg # %d"
#~ msgstr "Не удалось получить сообщение cslib # %d"

#~ msgid "Could not clear cslib messages"
#~ msgstr "Не удалось очистить сообщения cslib"

#~ msgid "CS-Library error:"
#~ msgstr "Ошибка CS-библиотеки:"

#~ msgid "fetch error at row %d"
#~ msgstr "получена ошибка в строке %d"

#~ msgid "CS_END_DATA on row %d; canceling"
#~ msgstr "CS_END_DATA в строке %d; отмена"

#~ msgid "%s: Error (%d)"
#~ msgstr "%s: Ошибка (%d)"

#~ msgid "Failed fetching # of columns"
#~ msgstr "Сбой получения числа столбцов"

#~ msgid "Counted %d columns"
#~ msgstr "Подсчитано %d столбцов"

# fe/fe_log.c:92
#~ msgid "%s could not allocate datafmt info"
#~ msgstr "%s Не удалось получить информацию о формате данных"

# fe/fe_log.c:92
#~ msgid "%s could not allocate data fields"
#~ msgstr "%s Не удалось получить поля данных"

#~ msgid "ct_describe on col # %d"
#~ msgstr "ct_describe на столбце # %d"

#~ msgid "ct_describe on col # %d failed"
#~ msgstr "сбой ct_describe на столбце # %d"

#~ msgid "ct_describe(%d): '%s', %d (%s)"
#~ msgstr "ct_describe(%d): \"%s\", %d (%s)"

# fe/fe_log.c:92
#~ msgid "could not allocate data holder"
#~ msgstr "не удалось получить место для хранения данных"

#~ msgid "illegal type detected, aborting"
#~ msgstr "обнаружен недопустимый тип, прерывание"

# fe/fe_log.c:92
#~ msgid "could not allocate field"
#~ msgstr "не удалось получить поле"

#~ msgid "Could not register gda-sybase provider implementation"
#~ msgstr "Не удалось зарегистрировать реализацию провайдера gda-sybase"

#~ msgid "GDA Datasource Access for primebase"
#~ msgstr "GDA доступ к источнику данных для primebase"

#~ msgid "Could not connect to '%s@%s:%s' with connparm '%s'"
#~ msgstr "Не удалось подключиться к \"%s@%s:%s\" с параметрами \"%s\""

#~ msgid "Not enough memory for primebase_Error structure."
#~ msgstr "Недостаточно памяти для структуры primebase_Error."

#~ msgid "%2ld: ERROR: %ld (%ld) : %s.\n"
#~ msgstr "%2ld: ОШИБКА: %ld (%ld) : %s.\n"

#~ msgid "Could not register PRIMEBASE provider implementation"
#~ msgstr "Не удалось зарегистрировать реализацию провайдера PRIMEBASE"

#~ msgid "GDA Datasource Access for Sybase/MS SQL"
#~ msgstr "GDA доступ к источнику данных для Sybase/MS SQL"

#~ msgid "Not enough memory for tds_Connection"
#~ msgstr "Недостаточно памяти для tds_Connection"

# fe/fe_connection.c:46
#~ msgid "Reconnected to %s"
#~ msgstr "Пересоединение с \"%s\""

#~ msgid "gda_tds_error_make(%p,%p,%p,%p)"
#~ msgstr "gda_tds_error_make(%p,%p,%p,%p)"

#~ msgid "Could not register gda-tds provider implementation"
#~ msgstr "Не удалось зарегистрировать реализацию провайдера gda-tds"

#~ msgid "%s: misconfigured DSN entry"
#~ msgstr "%s: неверная запись DSN"

#~ msgid "%s: data source '%s' not found\n"
#~ msgstr "%s: источник данных \"%s\" не найден\n"

#~ msgid "Could not register <PROVIDER-NAME> provider implementation"
#~ msgstr "Не удалось зарегистрировать реализацию провайдера <PROVIDER-NAME>"
